spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/SvnKitEnvironmentRepository.java

```diff
@@ -183,9 +183,9 @@ private File getSvnPath(File workingDirectory, String label) {
 		File svnPath = new File(workingDirectory, label);
 		if(!svnPath.exists()) {
 			svnPath = new File(workingDirectory, "branches" + File.separator + label);
-		}
-		if(!svnPath.exists()) {
-			svnPath = new File(workingDirectory, "tags" + File.separator + label);
+			if(!svnPath.exists()) {

```

Comment:
is the intention to return the branches or tags path if the original path does not exist?  what if both the branches and tags paths don't exist then we return the path to tags, is that intentional? 

I guess my question is should we only be setting `svnPath` if either the branches or tags directories exist, otherwise leave the behavior as it is today?